src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -122,6 +139,34 @@ public function getSerializedName(): ?string
         return $this->serializedName;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getSerializedNames(): array
+    {
+        return $this->serializedNames;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSerializedNameForGroups(array $groups = []): ?string
+    {
+        $defaultSerializedName = null;
+
+        foreach ($this->serializedNames as $serializedName => $groupsForSerializedName) {
+            if (empty($groupsForSerializedName)) {
```

Comment:
```suggestion
            if ($groupsForSerializedName) {
```

Reply:
if `empty()` is removed you should add `!`

Reply:
done